Kernel_23/doc/Kernel_23/Concepts/FunctionObjectConcepts.h

```diff
@@ -417,7 +417,7 @@ class BoundedSide_2 {
     \ref CGAL::ON_BOUNDARY, depending on where point `p` is relative to
     rectangle `r`.
   */
-  Bounded_side operator()(const Kernel::Iso_rectangle_2& r,
```

Comment:
Does not seem valid to me. Here we are not talking about the concept, but of the nested type `Iso_rectangle_2` from the `Kernel`

Reply:
You are right. But that mean that the links  like `Kernel::Point_2` with a link to the concept are wrong. Would be quite some work to remove them with a `%`.

Reply:
Best solution would be to rename `Kernel` namespace to `Kernel_concepts`

Reply:
What about `KernelObjects` / `KernelFunctors`?

Reply:
Why is it nost just   `operator()( Point_3)`, because we  are in the scope of `Kernel` ?

Reply:
That's a good point but you are not really in a Kernel scope. You define the requirement for a functor that could live outside of the Kernel (says Projection_traits for example). Kernel does not make sense and there should be nested types in the functor.

Reply:
`Kernel::Point_3` is the `Point_3` typedef defined in the `Kernel` concept, so it's well defined (?)